Formula/stout.rb

```diff
@@ -0,0 +1,22 @@
+require "language/go"
+
+class Stout < Formula
+  desc "Reliable static website deploy tool"
+  homepage "http://stout.is"
+  url "https://github.com/EagerIO/Stout/archive/v1.2.2.tar.gz"
+  sha256 "e04a44735c4c04ccce63e73d211d2a8c0232594b23c8dfc5c971c10d1aa72aaa"
+  depends_on "go" => :build
```

Comment:
As much as I appreciate compactness, er, this is overkill 😄. As per the conventions of other formulae, please leave a single empty line between the end of each block, i.e.

```ruby

sha256 "e04a44735c4c04ccce63e73d211d2a8c0232594b23c8dfc5c971c10d1aa72aaa"

depends_on "go" => :build
depends_on "godep" => :build

def install
  blah
end

test do
  blah
end
```